### ISSUE TITLE
Color contrast of tree view selected item and its border is less than 3:1

### ISSUE THREAD

**Issue by setaskin** [microsoft/vscode#151577]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.67.2
- OS Version: Windows 11

Steps to Reproduce:

1. Go to VS Code explorer 
2. Select an item on tree. 
3. Observe color contrast with selected item and its border

Expected : Color contrast of keyboard focus on the selected item should be than 3:1.

Actual :  Color contrast is less than 3:1.

<img width="1004" alt="image" src="https://user-images.githubusercontent.com/37163639/172743042-da6d4ba0-2f4c-4770-8f14-de72fd9ac9a6.png">

**Comment by setaskin** [comment]
Hi ,@isidorn would you help us with this accessibility issue.

**Comment by isidorn** [comment]
@setaskin thanks for opening this issue.
@misolori this somewhat makes sense. We should have a stronger contrast to make it more identifiable that a selected item is also focused. What do you think?

fyi @joaomoreno

**Comment by misolori** [comment]
This one is going to be tricky because we have an array of combinations of states for the border, but we can take a look:

<img width="521" alt="CleanShot 2022-06-09 at 10 00 29@2x" src="https://user-images.githubusercontent.com/35271042/172903308-f5f724c3-89e1-4828-bffe-94952243daa0.png">

**Comment by setaskin** [comment]
hi,  @isidorn and @misolori .  Thank you for looking into this.  We are (Live Share) very close to get our Grade C back . This is one of the last 3 bugs remaining that blocks our grade. If it can be prioritized that would be amazing. :)

**Comment by isidorn** [comment]
@misolori thanks for the investigation, and I like what path you took in that PR.

**Comment by curib** [comment]
Hi @misolori, thanks for working on this! Do you have an ETA we could share with the accessibility team? Thank you again!

**Comment by misolori** [comment]
@curib we're waiting on https://github.com/microsoft/vscode/pull/152260 to be approved & merged, so hopefully within the next week.

**Comment by setaskin** [comment]
@misolori , hi , accessibility team says it still fails on light theme. (Dark theme seems fine) They will post screen shots here. Has it been tested on light theme?

**Comment by fsteffi** [comment]
![image](https://user-images.githubusercontent.com/24580570/177777041-3222dbf6-36cc-4847-8701-1af9acbfc7d2.png)
in dark theme it works well


in light mode it fails 
![image](https://user-images.githubusercontent.com/24580570/177777108-e19e2406-f95a-4aa0-8651-a8a8505e89d5.png)

version used
Version: 1.69.0-insider (user setup)
Commit: 0b3574dcef8f35fec4ee4f83dc958c1f16ef6fce
Date: 2022-07-01T05:17:04.343Z
Electron: 18.3.5
Chromium: 100.0.4896.160
Node.js: 16.13.2
V8: 10.0.139.17-electron.0
OS: Windows_NT x64 10.0.22000
